legacy.html

```diff
@@ -0,0 +1,167 @@
+    <link rel="stylesheet" href="legacy.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.1.1/animate.min.css">
+
+    <!-- Character encoding and viewport-->
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <!-- Title for page-->
+    <title>Legacy</title>
+</head>
+
+<body class="bgimg">
+    <!-- Perfect dark logo-->
+    <div id="pdlogo" class="text-center p-3 pt-4">
+        <a href="https://en.wikipedia.org/wiki/Perfect_Dark">
+            <img src="https://www.theinitiative.com/wp-content/uploads/2021/07/pd-logo-white.png" width="200"
+                height="60">
+        </a>
+    </div>
+
+    <header class="sticky-top">
+        <!-- Nav bar -->
+        <nav class="navbar navbar-expand-sm navbar-dark p-3">
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
+                aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse pb-2" id="navbarSupportedContent">
+                <ul class="navbar-nav mx-auto">
+                    <li class="nav-item px-3">
+                        <a class="nav-link" href="#">HOME</a>
+                    </li>
+                    <li class="nav-item px-3">
+                        <a class="nav-link" href="#">GAMEPLAY</a>
+                    </li>
+                    <li class="nav-item px-3">
+                        <a class="nav-link" href="development_page.html">DEVELOPMENT</a>
+                    </li>
+                    <li class="nav-item px-3">
+                        <a class="nav-link" href="#">RECEPTION</a>
+                    </li>
+                    <li class="nav-item px-3 dropdown active">
+                        <a class="nav-link dropdown-toggle pdfont" href="#" id="navbarDropdownMenuLink" role="button"
+                            data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                            OTHER
+                        </a>
+                        <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
+                            <a class="dropdown-item" href="marketing_release.html">MARKETING AND RELEASE</a>
+                            <a class="dropdown-item pdfont" href="legacy.html">LEGACY</a>
+                            <a class="dropdown-item" href="remaster.html">REMASTER</a>
+                        </div>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+    </header>
```

Comment:
You need to update your header to the new one that our team uses for the website (you could find it in my code on Home Page)